client/data/media/use-update-media-mutation.js

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react';
+import { useMutation } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+export const useUpdateMediaMutation = ( queryOptions = {} ) => {
+	const mutation = useMutation(
+		( { siteId, mediaId, updates } ) =>
+			wp.req.post( `/sites/${ siteId }/media/${ mediaId }`, updates ),
```

Comment:
The original `updateMedia` action takes the `updates` object, merges it into the media item object that's already in state, and then sends the _full_ object to the REST endpoint. But here we're sending just the `updates`. Is that OK? Can the endpoint handle that?

Reply:
Yup, see https://github.com/Automattic/wp-calypso/pull/58604#discussion_r759223141